llama/config.py

```diff
@@ -37,6 +39,7 @@
     DATA_WAREHOUSE_HOST = os.getenv("ALMA_DATA_WAREHOUSE_HOST")
     DATA_WAREHOUSE_PORT = os.getenv("ALMA_DATA_WAREHOUSE_PORT")
     DATA_WAREHOUSE_SID = os.getenv("ALMA_DATA_WAREHOUSE_SID")
+    SENTRY_DSN = None
 
 
 def get_alma_api_key(parameter_name=None):
```

Comment:
I would add a new function to the end of `config.py` that we can call to confirm the Sentry integration is working. For example:
```
def check_sentry():
    if SENTRY_DSN:
        logger.info("Sending a Zero Division Error to Sentry")
        1 / 0
    else:
        logger.info("No Sentry DSN found")
```
And then for quick local verification, you can call that function from the main cli command (with the SENTRY_DSN env variable set) and view the exception on Sentry to confirm. Can't remember if we've talked about this, but at some future point we may want to add a cli command to check for all the appropriate config and SSM params that might also call this function so we can automatically verify everything is working on stage and prod when this app is deployed.

Reply:
Inadvertently sent a different error, but verified that it's sending exceptions to Sentry